### PR TITLE
new 8 NewsPage layout Kucher

### PR DESCRIPTION
**Pull request by MetalMari**

### CODE REVIEW

**Review by Maksson — APPROVED** [review]

**Comment by MetalMari** [comment]
File added by mistake must be deleted

**Comment by MetalMari** [comment]
Ok

**Review by s0fko — APPROVED** [review]

File: front-end/src/App.js [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
 
 function App() {
   return (
     <>
       <Auth_Header />
-      <li className='App'>
-        <MainContainer />
-        <Footer />
-      </li>
+      {/* <Header /> */}
```
Comment (vvelizariy) [inline front-end/src/App.js:10]: Is this comment required?

Comment (MetalMari) [reply front-end/src/App.js:10]: Yes, it's for "unauthorized user", but there isn't functionality for switching

**Review by Viktorika-p — APPROVED** [review]

**Pull request merged**